styles/main.css

```diff
@@ -207,46 +212,43 @@ header a:hover {
 }
 main {
     display: grid;
-    grid-template-columns: auto;
+    grid-template-columns: 1fr 1fr;
     column-gap: 2em;
     
 }
 .bio{
-    grid-column: span foo / 5;
+    grid-column: span 2;
     
 }
 .glossary{
-    display:grid;
-    grid-template-columns: auto auto auto auto;
-    grid-column: span foo / 4;
+    grid-column: span 2;
 }
 #useful-links{
-    display:grid;
-    grid-template-columns: auto auto;
-    grid-column: span foo / 4;
+    grid-column: span 2;
 }
 .gallery {
-    display:block;
-    grid-column: span foo / 5;
-    grid-column: 1 / 2; 
-    filter: grayscale(100%);
-    -webkit-filter: grayscale(100%);
+    grid-column: span 2;
+    width: 100%;

```

Comment:
The grid-column declaration makes the gallery span 2 columns.
Missing width declaration
The filter declarations should be applied to individual images